### PR TITLE
add recipe for pheasant

### PR DESCRIPTION
**Pull request by pkestene**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by pkestene** [comment]
There is a dependency over mkdocs >= 1.1.1 but only mkdocs 1.1 exists in conda-forge, so this MR must stall waiting an update of mkdocs recipe

File: recipes/pheasant/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Documentation tool for Markdown conversion by Jupyter client'
+  doc_url: https://pheasant.daizutabi.net/
+  dev_url: https://github.com/daizutabi/pheasant
+
+extra:
+  recipe-maintainers:
+    - daizutabi
```
Comment (synapticarbors) [inline recipes/pheasant/meta.yaml:40]: Please confirm that the co-maintainer agrees to be listed

File: recipes/pheasant/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0466da0a94eb2324dab43e4583b1251891adb012a6d264aaed42c7813cbfd07e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (synapticarbors) [inline recipes/pheasant/meta.yaml:15]: It looks like you need to add an entry for a python entry_point based on:

https://github.com/daizutabi/pheasant/blob/master/setup.py#L90

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

and then add a command test:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

File: recipes/pheasant/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+requirements:
+  host:
+    - python >3.7
+    - pip
+    - setuptools
+  run:
+    - python >3.7
```
Comment (synapticarbors) [inline recipes/pheasant/meta.yaml:23]: You will need the following under `run`:

https://github.com/daizutabi/pheasant/blob/master/setup.py#L80-L86

File: recipes/pheasant/meta.yaml (hunk 1 above)
Comment (pkestene) [reply recipes/pheasant/meta.yaml:40]: @daizutabi as the author of package pheasant, are you ok to be listed as a maintainer of the conda-forge package ?

File: recipes/pheasant/meta.yaml (hunk 3 above)
Comment (pkestene) [reply recipes/pheasant/meta.yaml:25]: @synapticarbors thanks for you inputs; I tried to address them in 5a95729bf5c3744885f356c837cb498f8502d8ef

File: recipes/pheasant/meta.yaml (hunk 1 above)
Comment (daizutabi) [reply recipes/pheasant/meta.yaml:50]: Yes, I'm OK.

File: recipes/pheasant/meta.yaml (hunk 3 above)
Comment (pkestene) [reply recipes/pheasant/meta.yaml:25]: @synapticarbors could you please review again ? I don't know to proceed from here.
Thank you.

**Pull request merged**